<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CylinderWidget</class>
 <widget class="QWidget" name="CylinderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Endpoint 1</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="endpoint1XLineEdit">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="endpoint1YLineEdit">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="4">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="endpoint1ZLineEdit">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Radius 1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="radius1LineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Endpoint 2</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="endpoint2XLineEdit"/>
   </item>
   <item row="2" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="endpoint2YLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="2" column="4">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="endpoint2ZLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Radius 1</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="radius2LineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Number of Facets</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="numFacetsLineEdit"/>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>(Around Circumference)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>endpoint1XLineEdit</tabstop>
  <tabstop>endpoint1YLineEdit</tabstop>
  <tabstop>endpoint1ZLineEdit</tabstop>
  <tabstop>radius1LineEdit</tabstop>
  <tabstop>endpoint2XLineEdit</tabstop>
  <tabstop>endpoint2YLineEdit</tabstop>
  <tabstop>endpoint2ZLineEdit</tabstop>
  <tabstop>radius2LineEdit</tabstop>
  <tabstop>numFacetsLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
